.github/workflows/test.yml

```diff
@@ -5,50 +5,50 @@ on:
     branches:
       - main
 jobs:
-  test-polyfill:
+  test-demitasse:
```

Comment:
OK, happy to keep the name the same. I renamed it because all of these test the polyfill, but we can always fix up the name later when there aren't a bunch of PRs queued up.